pkg/config/analysis/local/local.go

```diff
@@ -29,6 +29,5 @@ const (
 // AnalysisResult represents the returnable results of an analysis execution
 type AnalysisResult struct {
 	Messages          diag.Messages
-	SkippedAnalyzers  []string
```

Comment:
SkippedAnalyzers can be set when an analyzer requires an input that is not provided by any of the specified sources.  These cases are rare, but when they occur, this field is the only indication that the analyzer did not run and pass.  I can see how this should be more rare, now that we have arbitraryStore, but I'm nervous about missing an edge case, and reporting skipped analyzers as having passed...

Reply:
As discussed offline, I misunderstood this. Added this back. Thanks!